preprocess/analysis/handlers/routes.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask, jsonify, request, abort
+from language_analysis import analyze_entities, analyze_sentiment
+
+ERROR_MESSAGE = ('"header" or "content" field was either empty,' +
+                 'not included in form or has invalid values.')
+MAX_CHAR_LIMIT = 1000
+
+def is_valid(string):
+  return not (string is None or not string or len(string) > MAX_CHAR_LIMIT)
```

Comment:
You could do individual error messages for each of these issues to make things easier on your future self